reactor-netty-core/src/main/java/reactor/netty/channel/MicrometerChannelMetricsRecorder.java

```diff
@@ -86,9 +81,9 @@ public void recordDataReceived(SocketAddress remoteAddress, long bytes) {
 		String address = reactor.netty.Metrics.formatSocketAddress(remoteAddress);
 		DistributionSummary ds = MapUtils.computeIfAbsent(dataReceivedCache, address,
 				key -> filter(DistributionSummary.builder(name + DATA_RECEIVED)
```

Comment:
just to be sure: if my interpretation is correct from my other two comments, then this would have been equivalent (although concatenation of constants is more concise and prob more efficient):
```suggestion
				key -> filter(DistributionSummary.builder(name + DATA_RECEIVED)
				key -> filter(DistributionSummary.builder(ChannelMeters.DATA_RECEIVED.getName(name, DATA_RECEIVED))
```

Reply:
yes exactly